src/extra/imgui.cpp

```diff
@@ -883,7 +931,7 @@ template <CBType CT> struct Variable : public Base {
       } else {
         _variable_name = value.payload.stringValue;
       }
-      cleanup();
+      cleanupVariable();
```

Comment:
This is not so good actually, ParamVar better (Not blaming you, it's very old code originally :) )